src/main/resources/db/migration/common/V1_1__base_script.sql

```diff
@@ -415,7 +415,7 @@ IS 'used to record acceptable external locations, found in external_object_direc
 CREATE TABLE darts.hearing
 (hea_id                     INTEGER					   NOT NULL
 ,cas_id                     INTEGER                    NOT NULL
-,ctr_id                     INTEGER                    NOT NULL
+,ctr_id                     INTEGER
```

Comment:
Not sure about this change, to create a hearing we should always have a courtroom.